spec/expression.dd

```diff
@@ -91,12 +91,16 @@ $(H3 $(LNAME2 assignment_operator_expressions, Assignment Operator Expressions))
         a = cast(typeof(a))(a op b)
         --------------
 
-    except that:
+    except that
 
     $(UL
-        $(LI operand $(D a) is only evaluated once)
-        $(LI overloading $(I op) uses a different function than overloading $(I op)= does)
-        $(LI the left operand of $(D >>>=) does not undergo integral promotions before shifting)
+        $(LI operand $(D a) is only evaluated once,)
+        $(LI overloading $(I op) uses a different function than overloading $(I op)= does, and)
+        $(LI the left operand of $(D >>>=) does not undergo integral promotions before shifting.)
+    )
+    
+    $(P For user-defined types, assignment operator expressions can be overloaded separately from
+        the binary operator. Still the left operand must be an lvalue.
```

Comment:
I think actually, it *must* be overloaded separately. I tried just defining opBinary!"+", and += didn't work. i.e. there is no equivalent rewrite for user types.